models/keyword_condition.go

```diff
@@ -18,6 +18,7 @@ type Condition struct {
 // Map query string filter to where statement
 var ConditionMapper = map[string]string{
 	"keyword": "LOWER(keyword) LIKE LOWER('%%%s%%')",
+	"url":     "(LOWER(non_adword_links::text) LIKE '%%%[1]s%%' OR LOWER(top_position_adword_links::text) LIKE '%%%[1]s%%')",
```

Comment:
@gutakk One small question, does using `LOWER()` instead of `ILIKE` here make it better in terms of performance?

Reply:
This is the result using `EXPLAIN ANALYZE` for 60 rows data with following query
- Using LOWER LIKE
```sql
EXPLAIN ANALYZE SELECT * FROM keywords WHERE (LOWER(non_adword_links::text) LIKE '%wiki%' OR LOWER(top_position_adword_links::text) LIKE '%wiki%');

Seq Scan on keywords  (cost=0.00..7.80 rows=1 width=657) (actual time=0.186..1.391 rows=21 loops=1)
   Filter: ((lower((non_adword_links)::text) ~~ '%wiki%'::text) OR (lower((top_position_adword_links)::text) ~~ '%wiki%'::text))
   Rows Removed by Filter: 39
Planning Time: 0.975 ms
Execution Time: 1.489 ms
(5 rows)
```

- Using ILIKE
```sql
EXPLAIN ANALYZE SELECT * FROM keywords WHERE non_adword_links::text ILIKE '%wiki%' OR top_position_adword_links::text ILIKE '%wiki%';

Seq Scan on keywords  (cost=0.00..7.50 rows=1 width=657) (actual time=0.081..2.509 rows=21 loops=1)
   Filter: (((non_adword_links)::text ~~* '%wiki%'::text) OR ((top_position_adword_links)::text ~~* '%wiki%'::text))
   Rows Removed by Filter: 39
Planning Time: 0.262 ms
Execution Time: 2.550 ms
(5 rows)
```

And here is some more performance discussion
https://stackoverflow.com/questions/20336665/lower-like-vs-ilike